### PR TITLE
Week2 multiproc

### PR DESCRIPTION
**Pull request by Slavko12345**
Added the script that compares speeds of processing of csv file with 1 and 4 processes.
Please check.

### CODE REVIEW

**Review by truskovskiyk — COMMENTED** [review]
could you add your findings? how does multiprocessing help here? what is speedup?

**Comment by Slavko12345** [comment]
Added comparison timings to readme file. Speed up for 4 processes is 3.2 times.

File: multiprocessing/processing_comparison.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    
+    time_0 = time()
+    result_single = process_single_core(dataset)
+    time_1 = time()
+    result_multiple = process_multiple_cores(dataset, n_cores=4)
+    time_2 = time()
+    
+    assert result_single==result_multiple
+    
+    print("single processing time: ", time_1 - time_0)
```
Comment (truskovskiyk) [inline multiprocessing/processing_comparison.py:44]: is it part of another PR https://github.com/Slavko12345/course/pull/7#pullrequestreview-1127483871  ?

Comment (Slavko12345) [reply multiprocessing/processing_comparison.py:44]: Thanks for pointing it out, it was a mistakenly added file. Removed that file from PR #7.

**Pull request merged**